doc/how_to_guides/pilz_industrial_motion_planner/src/pilz_sequence.cpp

```diff
@@ -0,0 +1,366 @@
+
+  // Call the service and process the result
+  auto service_future = service_client->async_send_request(service_request);
+
+  // Function to draw the trajectory
+  auto const draw_trajectory_tool_path =
+      [&moveit_visual_tools,
+       jmg = move_group_interface.getRobotModel()->getJointModelGroup("panda_arm")](auto const& trajectories) {
+        for (const auto& trajectory : trajectories)
+        {
+          moveit_visual_tools.publishTrajectoryLine(trajectory, jmg);
+        }
+      };
+
+  // Wait for the result
+  std::future_status service_status;
+  do
+  {
+    switch (service_status = service_future.wait_for(std::chrono::seconds(1)); service_status)
+    {
+      case std::future_status::deferred:
+        RCLCPP_ERROR(LOGGER, "Deferred");
+        break;
+      case std::future_status::timeout:
+        RCLCPP_INFO(LOGGER, "Waiting for trajectory plan...");
+        break;
+      case std::future_status::ready:
+        RCLCPP_INFO(LOGGER, "Service ready!");
+        break;
+    }
+  } while (service_status != std::future_status::ready);
+
+  auto service_response = service_future.get();
+  if (service_response->response.error_code.val == moveit_msgs::msg::MoveItErrorCodes::SUCCESS)
+  {
+    RCLCPP_INFO(LOGGER, "Planning successful");
+
+    // Access the planned trajectory
+    auto trajectory = service_response->response.planned_trajectories;
+    draw_trajectory_tool_path(trajectory);
+    moveit_visual_tools.trigger();
+  }
+  else
+  {
+    RCLCPP_ERROR(LOGGER, "Planning failed with error code: %d", service_response->response.error_code.val);
+
+    rclcpp::shutdown();
+    return 0;
+  }
+
+  moveit_visual_tools.prompt("Press 'next' in the RvizVisualToolsGui window to continue the demo");
+
+  // [ --------------------------------------------------------------- ]
+  // [ ------------------ MoveGroupSequence Action ------------------- ]
+  // [ --------------------------------------------------------------- ]
+  // Plans and executes the trajectory
+
+  // MoveGroupSequence action client
+  using MoveGroupSequence = moveit_msgs::action::MoveGroupSequence;
+  auto action_client = rclcpp_action::create_client<MoveGroupSequence>(node, "/sequence_move_group");
+
+  // Verify that the action server is up and running
+  if (!action_client->wait_for_action_server(std::chrono::seconds(10)))
+  {
+    RCLCPP_ERROR(LOGGER, "Error waiting for action server /sequence_move_group");
+    return -1;
+  }
+
+  // Create a MotionSequenceRequest
+  moveit_msgs::msg::MotionSequenceRequest sequence_request;
+  sequence_request.items.push_back(item1);
+  sequence_request.items.push_back(item2);
+
+  // Create action goal
+  auto goal_msg = MoveGroupSequence::Goal();
+  goal_msg.request = sequence_request;
+
+  // Planning options
+  goal_msg.planning_options.planning_scene_diff.is_diff = true;
+  goal_msg.planning_options.planning_scene_diff.robot_state.is_diff = true;
+  // goal_msg.planning_options.plan_only = true;
```

Comment:
remove or uncomment?

Reply:
I added a "Uncomment to only plan the trajectory" comment since I think this option could be useful for someone